packages/ui/src/composables/useFixedBar.ts

```diff
@@ -0,0 +1,29 @@
+import { computed, ExtractPropTypes, Ref } from 'vue'
+
+export const useFixedBarProps = {
+  hideOnScroll: { type: Boolean, default: false },
+  fixed: { type: Boolean, default: false },
+  bottom: { type: Boolean, default: false },
+}
+
+export function useFixedBar (props: ExtractPropTypes<typeof useFixedBarProps>, isScrolledDown: Ref<boolean>) {
+  const isHiddenComputed = computed(() => isScrolledDown.value ? !!props.hideOnScroll : false)
+
+  const transformComputed = computed(() => {
+    if (!props.bottom && !isHiddenComputed.value) { return '' }
+    return props.bottom && isHiddenComputed.value ? 'translateY(100%)' : 'translateY(-100%)'
+  })
+
+  const positionComputed = computed(() => {
+    if (props.fixed) { return 'fixed' }
+    return isHiddenComputed.value ? 'absolute' : undefined
+  })
```

Comment:
```suggestion
  const positionComputed = computed(
    () => props.fixed ? 'fixed' : isHiddenComputed.value && 'absolute' || undefined
  )
```

Reply:
If `props.fixed` -> `fixed`

If `!props.fixed `-> boolean check `isHiddenComputed.value` and, if `true` -> `absolute`. Can't see the way there could be undefined.

May be you have meant double ternary? But it's not good practice.